Makefile

```diff
@@ -26,7 +26,7 @@ ubuntu-dev-setup:
 	sudo apt-get update && sudo apt-get install -y \
 	  build-essential gcc librados-dev librbd-dev
 
-build:osdsdock osdslet osdsapiserver osdsctl
+build:osdsdock osdslet osdsapiserver osdsctl osds_verify osds_unit_test osds_integration_test osds_e2eflowtest_build osds_e2etest_build 
```

Comment:
I'm afraid it would be not so proper to add these flags when building the source code, because users would complain it would take so much time every time building the code.